utils/partitions.js

```diff
@@ -35,7 +35,7 @@ exports.partition = function(dataset, testSetStart, testSetCount) {
  * @note code adapted from Heather Arthur:  https://github.com/harthur/classifier/blob/master/test/cross-validation/cross-validate.js
  */
 exports.partitions = function(dataset, numOfPartitions, callback) {
-	var shuffledDataset = _.shuffle(dataset);
+	// var shuffledDataset = _.shuffle(dataset);
```

Comment:
Why did you remove the shuffling? 

In cross-validation, it is common to shuffle the data in order to make sure that there is no bias in the partition.